config/routes.rb

```diff
@@ -30,6 +31,8 @@
 
     get '/files/:id', to: 'db_files#show'
 
+    get '/courses/:course_id/unused', to: 'invites#unused'
```

Comment:
This route is a bit weird. It doesn't give enough information. Just looking at this, I would think: "An endpoint for unused courses".

I think you should change it to `/courses/:course_id/invites/unused`